### PR TITLE
Fix overriding semantics messaging version in newer openjdk 8 versions

### PR DESCRIPTION
**Pull request by tobiasake**
### Applicable Issues
In later java 8 version class is loaded in reversed order and if external eiffel-semantic version jar files are provided then internal built-in eiffel-semantic version is used.
External eiffel-semantic version is configured with java flag:
java -Djava.ext.dirs=/path/to/eiffel-semantic-jar-files-dir/ -jar /path/to/generate-service-x.y.z.war

-Djava.ext.dirs is deprecated in Java 8 since many years ago and the -classpath should be used in newer Java versions, example:
java -classpath "/path/to/eiffel-semantic-protocol-jars/*:publish-service/target/publish-service-2.0.26.war" org.springframework.boot.loader.WarLaunch
 

### Description of the Change
Change so external provided eiffel-semantic version is used if provided with "-Djava.ext.dirs".

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:

### CODE REVIEW

**Review by RajuBeemreddy1 — APPROVED** [review]

**Review by z-sztrom — APPROVED** [review]

**Comment by tobiasake** [comment]
Same change is done in RemRem-Publish, please review my PR in RemRem-Publish as well:
https://github.com/eiffel-community/eiffel-remrem-publish/pull/252

**Comment by tobiasake** [comment]
Added documentation about how to run RemRem-Generate and how to override Eiffel-Protocol version.
Please check and review.
@RajuBeemreddy1 @z-sztrom 

If docs are Ok, I will add same docs to the RemRem-Publish patch.

File: wiki/markdown/run.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+
+## Run RemRem-Generate With Java Command
+
+1. Change to service directory: 
+`cd service`
+
+2. Execute maven package command to build the RemRem-Generate war file:
+`mvn package -DskipTests`
+
```
Comment (z-sztrom) [inline wiki/markdown/run.md:27]: Add somethink like
  3. cd ..
otherwise CWD is in service and the commands below don't work.

Comment (tobiasake) [reply wiki/markdown/run.md:26]: Fixed.

**Review by RajuBeemreddy1 — APPROVED** [review]

**Review by z-sztrom — APPROVED** [review]

**Pull request merged**